get-started/resources.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Educational resources"
+keywords: get started, setup, orientation, quickstart, intro, concepts, kubernetes, docker desktop
+description: Get started resources learning docker
+---
+
+Docker and the broader community of Docker experts have put together many different ways to get further training and hands-on experience with the Docker platform. Expand your understanding of Docker and Kubernetes with these additional free and paid resources.
+
+# Instructor-Led Training
+Docker and our Authorized Training Partners offer instructor-led courses around the world. Great for teams and organizations and also available for private sponsored classrooms, register for an upcoming class at [https://training.docker.com](https://training.docker.com).
+
+Courses offered include:
+* [Docker Fundamentals](https://success.docker.com/training/courses/docker-fundamentals)
+* [Docker for Enterprise Developers](https://success.docker.com/training/courses/docker-for-enterprise-developers)
+* [Docker for Enterprise Operations](https://success.docker.com/training/courses/docker-for-enterprise-operations)
+* [Docker Kubernetes Service](https://success.docker.com/training/courses/docker-kubernetes-service)
+* [Docker Troubleshooting and Support](https://success.docker.com/training/courses/docker-support-troubleshooting)
+
+\* Courses are fee-based
+
+# Hosted Labs 
+These self-paced and hands-on workshops use free, hosted environments ([Play with Docker](https://labs.play-with-docker.com/) and [Play with Kubernetes](https://labs.play-with-k8s.com/)) that do not require anything to be installed. Follow along and learn more about Docker and Kubernetes.
+
+* [Kubernetes Workshop](https://training.play-with-kubernetes.com/kubernetes-workshop/)
+* [Microservice containerization and orchestration](https://training.play-with-docker.com/microservice-orchestration/)
+
+\* Labs are free but require registration with a Docker ID
```

Comment:
Are all of the labs free or only the ones marked with an asterisk? I don't see any marked with an asterisk, so just wanted to clarify.

Reply:
see other reply